swc-config.js

```diff
@@ -16,8 +16,9 @@ exports.config = {
       plugins: [["@functionless/ast-reflection", {}]],
     },
   },
-  minify: true,
+  minify: false,
   sourceMaps: "inline",
+  inlineSourcesContent: false,
```

Comment:
I think this has an impact where identifiers that have been renamed cannot be re-mapped.